plugins/uast-kotlin-base/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+    kotlin("jvm")
+    id("jps-compatible")
+}
+
+dependencies {
+    implementation(kotlinStdlib())
+    implementation(project(":compiler:psi"))
+    implementation(project(":compiler:light-classes"))
+
+    // BEWARE: Uast should not depend on IDEA.
```

Comment:
@jsjeon Bad comment. What is "IDEA" in this context? Clearly, it's not intelliij modules cause you depend on intellij modules below but then I don't understand what it is

Reply:
This came from https://github.com/JetBrains/kotlin/commit/06af2a88f0f5860bebc45f0b366c0a7665912a03 and I believe "should not depend on" means not using `compile` or `implementation` gradle dependency, and "IDEA" in this context is `:idea:idea-core`

Yeah, it's not intellij modules, but intellij-core. Also, other dependencies below I think you mentioned were idea-frontend* modules.

Reply:
Sent out #4360. PTAL.